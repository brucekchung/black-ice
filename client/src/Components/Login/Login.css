.Login {
  background: black;
  color: white;
}

.Login input {
  display: block;  
  margin: 20px auto;
  width: 60%;
}

.Login button {
  margin: 20px auto;
}
