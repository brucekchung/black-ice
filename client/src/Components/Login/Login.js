import React, { Component } from 'react'
import './Login.css'
import Nav from '../Nav/Nav'

class Login extends Component {
  constructor() {
    super()
    this.state = {
      email: '',
      password: ''
    }
  }

  handleChange=(e)=> {
    const { name, value } = e.target;
    this.setState({[name]:value});
  }

  render() {
    return (
      <div className='Login'>
        <Nav />
        <input 
          className='email' 
          name='email' 
          onChange={this.handleChange} 
          placeholder='email' 
          value={this.state.email} />
        <input 
          className='password' 
          name='password' 
          onChange={this.handleChange} 
          placeholder='password' 
          value={this.state.password} />
        <button className='submit'>Enter</button>
      </div>
    )
  }
}

export default Login
