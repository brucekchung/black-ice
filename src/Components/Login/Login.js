import React, { Component } from 'react'
import './Login.css'

class Login extends Component {
  constructor() {
    super()
    this.state = {}
  }


  render() {
    return (
      <div>
        Login Component
      </div>
    )
  }
}

export default Login
