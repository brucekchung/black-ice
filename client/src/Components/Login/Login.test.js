import React from 'react'
import Login from './Login'
import { shallow } from 'enzyme'

describe('Login', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Login />)
  })

  it('should match snapshot upon load', () => {
    expect(wrapper).toMatchSnapshot()
  })

})