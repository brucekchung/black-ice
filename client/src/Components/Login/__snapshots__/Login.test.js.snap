// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should match snapshot upon load 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Login />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Nav />,
        <input
          className="email"
          name="email"
          onChange={[Function]}
          placeholder="email"
          value=""
/>,
        <input
          className="password"
          name="password"
          onChange={[Function]}
          placeholder="password"
          value=""
/>,
        <button
          className="submit"
>
          Enter
</button>,
      ],
      "className": "Login",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "email",
          "name": "email",
          "onChange": [Function],
          "placeholder": "email",
          "value": "",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "password",
          "name": "password",
          "onChange": [Function],
          "placeholder": "password",
          "value": "",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Enter",
          "className": "submit",
        },
        "ref": null,
        "rendered": "Enter",
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Nav />,
          <input
            className="email"
            name="email"
            onChange={[Function]}
            placeholder="email"
            value=""
/>,
          <input
            className="password"
            name="password"
            onChange={[Function]}
            placeholder="password"
            value=""
/>,
          <button
            className="submit"
>
            Enter
</button>,
        ],
        "className": "Login",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "email",
            "name": "email",
            "onChange": [Function],
            "placeholder": "email",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "password",
            "name": "password",
            "onChange": [Function],
            "placeholder": "password",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Enter",
            "className": "submit",
          },
          "ref": null,
          "rendered": "Enter",
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
